packages/plugins/src/safe/utils/anonAadhaar/AnonAadhaar.sol

```diff
@@ -0,0 +1,77 @@
+//SPDX-License-Identifier: Unlicense
```

Comment:
I think these files can be imported from [Anon Aadhaar npm package](https://www.npmjs.com/package/@anon-aadhaar/contracts) instead of copied to here?

Reply:
Unfortunately, no, as I've made slight changes to this one and Verifier in order to get around opcode issues